modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/SW360ComponentClient.java

```diff
@@ -41,7 +44,9 @@ public SW360Component getComponent(String componentId, HttpHeaders header) throw
                 new ParameterizedTypeReference<Resource<SW360Component>>() {});
 
         if (response.getStatusCode() == HttpStatus.OK) {
-            return response.getBody().getContent();
+            return Optional.ofNullable(response.getBody())
+                    .orElseThrow(() -> new AntennaException("Body was null"))
```

Comment:
I think the exception message is not helpful enough..

Reply:
One sees in which line and in which function the exception was thrown. Isn't that enough?

Reply:
What about the `componentId`?

Reply:
should be fixed by: https://github.com/eclipse/antenna/compare/fc80ca9edee7309d10ea152d907a701546158c32..67b9c99cb3a76cf8a7433afaa3c811268322b680